ocr_service/api/endpoints/extraction.py

```
@@ -0,0 +1,293 @@
+
+# Models
+class ImageURLRequest(BaseModel):
+    image_url: str
+
+class PDFURLRequest(BaseModel):
+    pdf_url: str
+
+# URL Handler class - simplified for Cloudinary only
+class URLHandler:
+    @staticmethod
+    def is_cloudinary_url(url):
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `is_cloudinary_url`

(ANN205)